sdk/media/azure-media-livevideoanalytics-edge/azure/media/livevideoanalytics/edge/_generated/_version.py

```diff
@@ -0,0 +1,9 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = "1.0"
```

Comment:
@lmazuel, do we have a policy on what version to use for internal generated packages? Technically, I don't think it matters (it isn't part of the public API surface area), but I wasn't sure if we use it for any kind of internal bookkeeping.